package.json

```diff
@@ -25,7 +25,9 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "pretest": "yarn run lint",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache"
+    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
```

Comment:
I would recommend this be adjusted

Personally I would hope to see these new coveralls scripts namespaced with `test`. 

Barring that, I would at least hope that there weren't testing-specific scripts both namespaced under `test` (like `test:debug`) and not namespaced under `test` (like added scripts `coverage` and `coveralls`).

See note below for a possible motivation for removing the `coverage` script

Reply:
Thanks @SherSpock I just added a coma on this line. `test.debug` was added on an earlier PR so as to help with running the debugger for using breakpoints. But maybe something we may need to review the naming.

Reply:
The `test:debug` script is outside the scope of anything but merely observational and supporting comments for my review. I stand by my opening statement on this review -- the one critical thing I'm concerned about is the `.coveralls.yml`. My opinion here is that Cypress doesn't deserve to be a first class citizen in the top-level scripts namespace, nor does it deserve to be excluded from the `test` namespace. The `test` namespace is actually the most logical currently existent namespace for it to live in.

Reply:
@SherSpock we have discussed this in the #weekend-collab and decided to remove the jest.config.js file since the test coverage is currently generated by yarn test --coverage and not with the jest.config.js set up. We will create a ticket to configure the jest configuration.